CefSharp.OutOfProcess.BrowserProcess/BrowserProcessHandler.cs

```diff
@@ -124,5 +134,48 @@ void IOutOfProcessClientRpc.SetFocus(int browserId, bool focus)
 
             browser?.GetBrowserHost().SetFocus(focus);
         }
+
+        /// <summary>
+        /// Update Request Context Preferences of the browser.
+        /// </summary>
+        /// <param name="browserId">The browser id.</param>
+        /// <param name="preferences">The preferences.</param>
+        void IOutOfProcessClientRpc.UpdateRequestContextPreferences(int browserId, IDictionary<string, object> preferences)
+        {
+            var browser = _browsers.FirstOrDefault(x => x.Id == browserId);
```

Comment:
If the browser is null then this should fail fast.

Reply:
moved ui-thread code into if-body